cypress/e2e/awx/resources/inventories.cy.ts

```diff
@@ -53,81 +52,67 @@ describe('Inventories Tests', () => {
           cy.hasDetail(/^Organization$/, organization.name);
           cy.hasDetail(/^Enabled options$/, 'Prevent instance group fallback');
           cy.clickPageAction('delete-inventory');
           cy.get('#confirm').click();
           cy.clickButton(/^Delete inventory/);
-          //Add assertion verifying that the inventory has now been deleted- including verifying the 204 statusCode and
-          //filtering a list to show no results
           cy.verifyPageTitle('Inventories');
         });
 
         it('can edit an inventory from the list view and assert info on details page', () => {
-          //Refactor this test to match the updated test case and improve the assertions
           cy.navigateTo('awx', 'inventories');
           cy.filterTableBySingleSelect('name', inventory.name);
           cy.get(`[data-cy="row-id-${inventory.id}"]`).within(() => {
             cy.get('[data-cy="edit-inventory"]').click();
           });
           cy.multiSelectByDataCy('instance-group-select-form-group', [instanceGroup.name]);
-          cy.contains('button', 'Save inventory').click(); //Add an interception call for the edited inventory
+          cy.contains('button', 'Save inventory').click();
           cy.verifyPageTitle(inventory.name);
-          //Add assertions to verify the updated information is reflecting on the details screen of the edited inventory
           cy.hasDetail(/^Instance groups$/, instanceGroup.name);
         });
 
         it('can edit an inventory from the details view and assert info on details page', () => {
-          //Refactor this test to match the updated test case and improve the assertions
           cy.navigateTo('awx', 'inventories');
           cy.filterTableBySingleSelect('name', inventory.name);
           cy.clickTableRowLink('name', inventory.name, { disableFilter: true });
           cy.verifyPageTitle(inventory.name);
-          //Add more assertions to verify that the user is now on the details screen and to assert the inventory's original info
           cy.clickButton(/^Edit inventory/);
           cy.selectDropdownOptionByResourceName('labels', label.name);
           cy.dataEditorTypeByDataCy('variables', 'remote_install_path: /opt/my_app_config');
-          cy.contains('button', 'Save inventory').click(); //Add an interception call for the edited inventory
+          cy.contains('button', 'Save inventory').click();
           cy.verifyPageTitle(inventory.name);
-          //Add assertions to verify the updated information is reflecting on the details screen of the edited inventory
-          cy.assertMonacoTextField('remote_install_path: /opt/my_app_config'); //Refactor this line to use the updated custom command
+          cy.assertMonacoTextField('remote_install_path: /opt/my_app_config');
           cy.hasDetail(/^Labels$/, label.name);
         });
 
         it('can copy an inventory on the details view and assert that the copy has been successful', () => {
```

Comment:
intermittent failures on this one as well as `it('can delete an inventory from the inventory list toolbar', ()`
with error
```
get#filter
0
8
assertexpected #filter not to be disabled
```